packages/experimental/Tabs/src/TabsItem.types.win32.ts

```diff
@@ -0,0 +1,63 @@
+import * as React from 'react';
+import { ViewProps} from 'react-native';
+import { TextProps } from '@fluentui-react-native/experimental-text';
+import { IFocusable, IWithPressableOptions, IWithPressableEvents } from '@fluentui-react-native/interactive-hooks';
+import type { IViewWin32Props } from '@office-iss/react-native-win32';
+import { IconProps } from '@fluentui-react-native/icon';
+import { TabsItemTokens, TabsItemState } from './TabsItem.types';
+
+type IconSourcesType = number | string | IconProps;
+
+export interface TabsItemProps extends Omit<IWithPressableOptions<IViewWin32Props>, 'onPress'>   {
```

Comment:
Are the Tokens meant to be missing from this version?

Reply:
Yes, because those aren't win32 specific. So when the engineer is coding, they can import the tokens from the general types file and the specific win32 ones from this file. This is the way the Shimmer control seemed to do it.

Reply:
Yeah I think TabsItemTokens gets imported from the base (non-win32) file on line 59